etc/templates/profile.template

```diff
@@ -1,6 +1,21 @@
 # Firejail profile for PROGRAM_NAME
 # Description: DESCRIPTION
 # This file is overwritten after every install/update
+#
+# --- CUT HERE ---
+# This is a generic template to help you with creation of profiles
+# for new programs. PRs welcome at https://github.com/netblue30/firejail/
+#
+# Rules to follow:
+#  - make the profile as restrictive as possible while still keeping the program useful
+#    (e. g. a program that is unable to save user's work is considered a bad practice)
+#  - test profile for couple days before raising a pull request
+#  - keep the sections structure, use a single empty line as a separator
```

Comment:
the structure is

HEADER
COMMENTS
NOBLACKLISTS
BLACKLISTS
DISABLE INCLUDES
MKDIRS
WHITELISTS
WHITELIST INCLUDES
OPTIONS (no*)
PRIVATE OPTIONS (private-*)
SPECIAL OPTIONS (mdwx, noexec, read-only)
REDIRECT INCLUDES

Reply:
Thank you, will add

Reply:
@jose1711 @SkewedZeppelin `grep "^blacklist" --context=5 /etc/firejail/*.profile`
profile with `blacklist` and `noblacklist` have `blacklist` above `noblacklist`:
…
COMMENTS
BLACKLISTS
NOBLACKLISTS
DISABLE INCLUDES
…

Reply:
Also what means COMMENTS? the comments in the HEADER?

Reply:
To be complete:
PRIVATE OPTIONS (disable-mnt, private-*)
SPECIAL OPTIONS (mdwx, noexec, read-only, join-or-start)